app/controllers/users_controller.rb

```diff
@@ -56,12 +60,18 @@ def update
     if (params[:password].blank? && !current_user?(@user))
       params.delete(:password)
       params.delete(:password_confirmation)
+      # bcrypt gem is forcing net id user to have a password, have to bypass for now
+      if @user.password_digest.blank?
+        @user.update_attribute(:privilege, user_params[:privilege])
+        redirect_to @user and return
+      end
```

Comment:
this is definitely a hack. Only updates the privilege of a NET ID account, and skips the other attributes. the bcrypt gem is giving me trouble by enforcing passwords behind the scenes @boyangles